### PR TITLE
1360 - Bulk postArtifacts calls added to ExtractRegistry

### PR DESCRIPTION
**Pull request by kellykelly3**

### CODE REVIEW

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java [hunk 1]
```diff
@@ -873,20 +865,22 @@ private boolean parseAutopsyPluginOutput(String regFilePath, AbstractFile regFil
             } // for
             return true;
         } catch (FileNotFoundException ex) {
-            logger.log(Level.WARNING, String.format("Error finding the registry file: %s", regFilePath), ex); //NON-NLS
+            logger.log(Level.SEVERE, String.format("Error finding the registry file: %s", regFilePath), ex); //NON-NLS
```
Comment (dannysmyda) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java]: Why have these been changed to severe? This method is far too hard to read, but these seem related to bad data which I would think are warnings.

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java [hunk 2]
```diff
@@ -721,7 +718,7 @@ private boolean parseAutopsyPluginOutput(String regFilePath, AbstractFile regFil
                                                 itemMtime /= MS_IN_SEC;
                                             }
                                         } catch (ParseException ex) {
-                                            logger.log(Level.WARNING, "Failed to parse epoch time for installed program artifact.", ex); //NON-NLS
+                                            logger.log(Level.SEVERE, "Failed to parse epoch time for installed program artifact.", ex); //NON-NLS
```
Comment (dannysmyda) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java]: This seems like a malformed data exception, why is this logged as severe?

Comment (kellykelly3) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java]: They were originally SEVERE, I changed them to WARNING and I was asked by Richard to put them back the way they were.

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java (hunk 1 above)
Comment (kellykelly3) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java]: They were originally SEVERE, I changed them to WARNING and I was asked by Richard to put them back the way they were.

Comment (rcordovano) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java]: @dannysmyda and @kellykelly3, I was referring to TskCoreExceptions that I believed indicated an INSERT into the case database to make a ready to go artifact had failed. IN other words, my request covered situations where the analysis result was complete and in hand, but we were unable to add it to the case.

**Pull request merged**